### PR TITLE
Feat/id porten

### PR DESCRIPTION
**Pull request by amundlrohne**

### CODE REVIEW

File: apps/api/src/routes/CitizenRoutes.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        return res.json(result);
+    } catch (error) {
+        next(error);
+    }
+});
+
+citizenRoutes.post('/', authenticated, async (req, res, next) => {
+    try {
+        const citizenService = new CitizenService(database);
+        req.body.registered = true;
```
Comment (perkynades) [inline apps/api/src/routes/CitizenRoutes.ts:38]: Should the data be modified when it gets to the route? Feel like this goes against the normal flow of data objects

File: apps/api/src/routes/UserRoutes.ts [hunk 2]
```diff
@@ -1,9 +1,8 @@
-import { plainToInstance } from 'class-transformer';
 import { Router } from 'express';
 import { database } from '../loaders';
 import { User } from '../models/user/UserEntity';
 import { IUserSearch, UserService } from '../services';
-import { adminAuthenticated, authenticated, navAuthenticated } from './middleware/authenticationHandler';
+import { adminAuthenticated } from './middleware/authenticationHandler';
 
 const userRoutes = Router();
```
Comment (perkynades) [inline apps/api/src/routes/UserRoutes.ts]: Maby this should be renamed to "adminRoutes", as now it is only handling routes for admins

File: apps/frontend/src/api/mutations/mutateCitizen.ts [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+import { ValidationError } from 'class-validator';
+import { mutate } from 'swr';
+import { poster, putter } from '../operations';
+
+export const createCitizen = async (citizen: ICitizen) => {
+    const { data, error } = await poster<ICitizen>('/api/citizen', citizen);
+
+    if (error?.response?.status === 406) {
+        return { response: data, validationErrors: JSON.parse(error.response.data.message) as ValidationError[] };
+    }
```
Comment (perkynades) [inline apps/frontend/src/api/mutations/mutateCitizen.ts]: Consider createing seperate method as it is used for multiple api-hooks

**Review by perkynades — APPROVED** [review]
Very good work, consider switching the naming on files from "User" to "Admin" where the file is responsible for handling Admin stuff

File: apps/api/src/routes/CitizenRoutes.ts (hunk 1 above)
Comment (amundlrohne) [reply apps/api/src/routes/CitizenRoutes.ts:38]: Probably not, this solution is kind of temporary as we don't have the final design for the registration process. I will make an issue to move this to the service class.

File: apps/api/src/routes/UserRoutes.ts (hunk 2 above)
Comment (amundlrohne) [reply apps/api/src/routes/UserRoutes.ts]: This file can actually be deleted, as it's no longer in use. 😅

File: apps/frontend/src/api/mutations/mutateCitizen.ts (hunk 3 above)
Comment (amundlrohne) [reply apps/frontend/src/api/mutations/mutateCitizen.ts]: Will create an issue on this as the precedence was introduced before this PR.

**Pull request merged**